packages/flutter/test_fixes/widgets.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
```

Comment:
We should probably have a separate test that import widgets. dart (and maybe cupertino.dart) instead of material.dart? See above.

Reply:
Ah I see what you mean, thanks! I'll update.